system76driver/util.py

```diff
@@ -52,6 +52,18 @@ def dump_logs(base):
     fp = open(path.join(base, 'journalctl'), 'xb')
     SubProcess.check_call(['journalctl', '--since', 'yesterday'], stdout=fp)
 
+    fp = open(path.join(base, 'fstab'), 'xb')
+    SubProcess.check_call(['cat', '/etc/fstab'], stdout=fp)
+
+    fp = open(path.join(base, 'sources.list'), 'xb')
+    SubProcess.check_call(['cat', '/etc/apt/sources.list'], stdout=fp)
+
+    fp = open(path.join(base, 'sources.list.d-list'), 'xb')
+    SubProcess.check_call(['ls', '/etc/apt/sources.list.d/'], stdout=fp)
+
+    fp = open(path.join(base, 'sources.list.d-content'), 'xb')
+    SubProcess.check_call(['cat', '/etc/apt/sources.list.d/*'], stdout=fp)
```

Comment:
Did you test this? I don't believe globs (the asterisk) will work here

Instead consider using cp -r to copy the sources.list.d directory over

Reply:
That should have been removed since I did see that it didn't work. I'm correcting it now.